.github/workflows/test.yml

```diff
@@ -20,6 +20,22 @@ jobs:
           git branch --track main origin/master
       - name: Install dependencies
         uses: ./.github/workflows/dependencies-install
+
+      - name: Store Playwright's Version
+        run: |
+          PLAYWRIGHT_VERSION=$(npx playwright -V | awk '{ print $2 }')
+          echo "Playwright's Version: $PLAYWRIGHT_VERSION"
```

Comment:
Do we need to use `npx` here? Can`t we run `playwright -V` only? Because npx uses global `node:modules` I guess.

Reply:
Output of `yarn playwright -V`:
```
run
/Users/arbeit/Projects/elements/node_modules/.bin/playwright
1.41.2
in
```

Output of `npx playwright -V`:
```
1.41.2
```

So I would keep `npx` for now